Packages/com.bci4kids.bciessentials/Runtime/Scripts/StimulusObjects/StimulusEffects/TransformScaleEffect.cs

```diff
@@ -7,6 +7,10 @@ public class TransformScaleEffect: StimulusEffect
         [SerializeField]
         [Tooltip("Value to multiply or divide the local scale value by.")]
         private float _scaleValue = 1;
+
+        [SerializeField]
```

Comment:
The ganges on this file seem to be additional to the space aware multiflash. I think that is fine, just wanted to mention in case you'd like to put them as a separate PR